lib/pluginmanager/package.rb

```diff
@@ -0,0 +1,67 @@
+class LogStash::PluginManager::Package < LogStash::PluginManager::Command
+
+  option "--tgz", :flag, "compress package as a tar.gz file", :default => false
+  option "--zip", :flag, "compress package as a zip file", :default => false
+  option "--clean", :flag, "clean up the generated dump of plugins", :default => false
+  option "--override", :flag, "Override a previously generated package file", :default => false
```

Comment:
I don't think we need an `--override` flag. it think we can simply refuse to overwrite a package file and tell the user to delete it if they want to generate a new one?